streaming/src/main/scala/io/gearpump/streaming/appmaster/TaskManager.scala

```diff
@@ -115,8 +115,9 @@ private[appmaster] class TaskManager(
     case Event(TasksLaunched, state) =>
       // We will track all launched task by message RegisterTask
       stay
-    case Event(TasksChanged, state) =>
-      // We will track all changed task by message TaskChanged.
+    case Event(TasksChanged(tasks), state) =>
+      tasks.foreach(task =>state.taskChangeRegistry.taskChanged(task))
```

Comment:
I'd like to keep it as it is so that it is easier verify whether this handler work or not in unit test.